packages/linter/src/executors/eslint/lint.impl.ts

```diff
@@ -63,6 +66,23 @@ export default async function run(
     throw new Error('ESLint must be version 7.6 or higher.');
   }
 
+  if (printConfigForFile) {
+    try {
+      const fileConfig = await eslint.calculateConfigForFile(
+        printConfigForFile
+      );
+      console.log(JSON.stringify(fileConfig, null, ' '));
+      return {
+        success: true,
+      };
+    } catch (err) {
+      console.error(err);
+      return {
+        success: false,
+      };
+    }
+  }
```

Comment:
We don't need to mutate the `options` above, just use it here. Since all of the branches return we are safe.

```suggestion
  if (options.printConfig) {
    try {
      const fileConfig = await eslint.calculateConfigForFile(
        options.printConfig
      );
      console.log(JSON.stringify(fileConfig, null, ' '));
      return {
        success: true,
      };
    } catch (err) {
      console.error(err);
      return {
        success: false,
      };
    }
  }
```